iroha/src/config.rs

```diff
@@ -186,6 +191,55 @@ impl ConfigurationBuilder {
     }
 }
 
+/// Parses string formatted as "[address1, address2, ...]" into `Vec<PeerId>`.
+fn parse_trusted_peers(
```

Comment:
Initially we had only key value map in configuration. Now we parse arrays and objects. I wonder if we should implement a full json parser at this stage. Otherwise we might end up with a lot of duplicate code, which will be hard to change for some new parameters in config.

Reply:
Agree, I think we will end up with it, are you agree?

Reply:
Yes, let's create an issue for that

Reply:
https://soramitsu.atlassian.net/browse/RH2-96